### PR TITLE
Bump version to v0.1.8

### PR DESCRIPTION
**Pull request by charliermarsh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## 0.1.8
+
+### Preview features
```
Comment (dhruvmanila) [inline CHANGELOG.md:5]: Do we want to start including the blog post here in a summary? Cross linking between the blog post and changelog might be useful.

Comment (charliermarsh) [reply CHANGELOG.md:5]: Yeah good call.

**Review by BurntSushi — APPROVED** [review]
LGTM!

**Review by zanieb — APPROVED** [review]
<3

**Pull request merged**